doc/api/assert.md

```diff
@@ -1244,6 +1244,25 @@ assert.throws(throwingFirst, /Second$/);
 Due to the confusing notation, it is recommended not to use a string as the
 second argument. This might lead to difficult-to-spot errors.
 
+## assert.undefinedProperty
+<!-- YAML
+added: REPLACEME
+-->
+
+This property can be used in combination with `assert.throws()` and
+`assert.rejects()` when using the object notation to validate the actual error.
+Passing it along makes sure the property is not set on the error object.
```

Comment:
`Passing it along makes sure` -> ...something else...

Maybe: `Passing it as a property on the expected object makes sure`?

Or: `Passing it makes sure`...

Or: ```Using it causes `assert.throws()` to check that```....

@nodejs/documentation